boxoffice/models/line_item.py

```diff
@@ -100,6 +99,10 @@ def calculate(cls, line_item_dicts, coupons=[]):
     def confirm(self):
         self.status = LINE_ITEM_STATUS.CONFIRMED
 
+    @property
+    def current_assignee(self):
+        return self.assignees.filter(Assignee.current == True).one_or_none()
```

Comment:
This can be a relationship: `assignee = db.relationship(Assignee, primaryjoin=Assignee.line_item == self and Assignee.current == True)`

Don't use `current_assignee` -- we want to imply that there can only be one assignee and the rest are historical (and hence not 'assignees')